lib/train/platforms/detect/specifications/os.rb

```diff
@@ -461,7 +461,7 @@ def self.load
       plat.name('mac_os_x').title('macOS X').in_family('darwin')
           .detect {
             cmd = unix_file_contents('/System/Library/CoreServices/SystemVersion.plist')
-            @platform[:uuid_command] = "system_profiler SPHardwareDataType | awk '/UUID/ { print $3; }'"
+            self.uuid_command = "system_profiler SPHardwareDataType | awk '/UUID/ { print $3; }'"
```

Comment:
should this be `@platform.uuid_command`? I think self works but it is a bit hard to figure out who is self when you'r passing around these procs.